MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/MuonFitPropertyBrowser.h

```diff
@@ -149,6 +190,32 @@ private slots:
   std::string m_simultaneousLabel;
   QtProperty *m_normalization;
   mutable QStringList m_normalizationValue;
```

Comment:
`m_normalizationValue` and a few of the other variables below it are set to mutable